### DISCUSSION TITLE
How to make fzf-lua respect FZF_DEFAULT_OPTS_FILE

### DISCUSSION THREAD

**Discussion by astier** [ibhagwan/fzf-lua#1300] [General]
Since [fzf 0.47](https://github.com/junegunn/fzf/releases/tag/0.47.0) there is `$FZF_DEFAULT_OPTS_FILE` to allow managing default options in a file. Before that I was using `fzf_args = vim.env.FZF_DEFAULT_OPTS`. How to make `FZF_DEFAULT_OPTS_FILE` work with fzf-lua?

**Comment by ibhagwan** [comment]
Fzf-lua respects both by default so you don’t need to do anything special to have it respected, however it does override arguments, like `--info` by default so you’ll have to override these options, you can probably use `tr` to achieve the same effect if you wanted, see https://github.com/ibhagwan/fzf-lua/discussions/1289#discussioncomment-9844652
